_docs/Definition-of-Done.md

```diff
@@ -1,14 +1,14 @@
 # Definition of Done
 
--   All tasks are updated and closed
--   No high priority defects/bugs are on that user story
--   Code reviewed by one other team member (e.g. Checked by Bill)
--   All acceptance criteria have been met
--   Deployed to Dev
--   (If implemented) Pass automated UI testing
--   (If not implemented above) Do a manual test on all supported commands (at least one for each LUIS intent) against Dev
--   Complex code is documented (removing technical debt)
--   Deployed to Production
--   Add description about the changes for the PBI
--   Tick off the definition of done tasks in the PBI
--   (Optional) A Done Video, particularly for PBI with multiple stages
+- All tasks are updated and closed
+- No high priority defects/bugs are on that user story
+- Code reviewed by one other team member (e.g. Checked by Bill)
+- All acceptance criteria have been met
+- Deployed to Dev
+- (If implemented) Pass automated UI testing
+- (If not implemented above) Please ensure that you do a manual test and meet the [pre-release requirements](./Sanity-Testing.md)
```

Comment:
I think pre-release requirements are mandatory so I would remove (If not implemented above)